templates/ocs-deployment/StorageClass.yaml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: {{ sc_name | default('ocsci-storageclass') }}
+provisioner: ceph.rook.io/block
+parameters:
+  blockPool: ocsci-cephblockpool
+  # Specify the namespace of the rook cluster from which to create volumes.
+  # If not specified, it will use `rook` as the default namespace of the cluster.
+  # This is also the namespace where the cluster will be
+  clusterNamespace: {{ cluster_namespace | default('openshift-storage') }}
```

Comment:
This should always be `openshift-storage` as `clusterNamespace` defines where ceph cluster is running.

Reply:
Done